### PR TITLE
Create example: shot at an angle to the horizon

### PR DESCRIPTION
**Pull request by Voldoned**

### CODE REVIEW

File: examples/dynamics/shot_at_an_angle_to_the_horizon.py [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+from symplyphysics import print_expression
+from symplyphysics.laws.kinematic import planar_projection_is_cosine as projection_velocity
+from symplyphysics.laws.dynamics import kinetic_energy_from_mass_and_velocity as kinetic_energy
+
+# The initial velocity of the bullet is 600 m/s, its mass is 10 g.
+# At what angle to the horizon did it fly out of the muzzle of the gun,
+# if its kinetic energy at the highest point of the trajectory is 450 J?
+
+start_velocity = Symbol("start_velocity")
+kinetic_energy_in_pick = Symbol("kinetic_energy_in_pick")
```
Comment (blackyblack) [inline examples/dynamics/shot_at_an_angle_to_the_horizon.py:11]:
```suggestion
kinetic_energy_at_peak = Symbol("kinetic_energy_at_peak")
```

File: examples/dynamics/shot_at_an_angle_to_the_horizon.py [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+angle_of_shot_equation = solve(kinetic_energy_equation, angle_of_shot, dict=True)[-1][angle_of_shot]
+answer = Eq(angle_of_shot, angle_of_shot_equation)
+print(f"Total angle of shot equation:\n{print_expression(answer)}")
+
+angle_of_shot_rad = angle_of_shot_equation.subs({
+    start_velocity: 600,
+    mass_of_bullet: 0.01,
+    kinetic_energy_in_pick: 450
+})
+print(f"Angle of shot is: {angle_of_shot_rad} rad")
```
Comment (blackyblack) [inline examples/dynamics/shot_at_an_angle_to_the_horizon.py:38]: Calculate degrees. Nobody likes radians.

File: examples/dynamics/shot_at_an_angle_to_the_horizon.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
...
+mass_of_bullet = Symbol("mass_of_bullet")
+
+angle_of_shot = Symbol("angle_of_shot")
+
+velocity_projection_equation = projection_velocity.law.subs(({
+    projection_velocity.vector_length: start_velocity,
+    projection_velocity.vector_angle: angle_of_shot
+})).rhs
+
+kinetic_energy_equation = kinetic_energy.law.subs({
```
Comment (blackyblack) [inline examples/dynamics/shot_at_an_angle_to_the_horizon.py:21]: Explain why we use horizontal projection for kinetic energy.

**Review by blackyblack — APPROVED** [review]

File: examples/dynamics/shot_at_an_angle_to_the_horizon.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
+from sympy import solve, Symbol, Eq
```
Comment (blackyblack) [inline examples/dynamics/shot_at_an_angle_to_the_horizon.py:1]: It is recommended to have this line at the beginning of each example: `#!/usr/bin/env python3`

File: examples/dynamics/shot_at_an_angle_to_the_horizon.py [hunk 5]
```diff
@@ -15,6 +16,10 @@
 
 angle_of_shot = Symbol("angle_of_shot")
 
+# At the highest point of the trajectory, the bullet velocity modulus is equal to
```
Comment (blackyblack) [inline examples/dynamics/shot_at_an_angle_to_the_horizon.py:19]: It's wrong. At the highest point there is no vertical component of the velocity, therefore we can use only horizontal component, which is projection of initial velocity on the horizon.

File: examples/dynamics/shot_at_an_angle_to_the_horizon.py (hunk 4 above)
Comment (Voldoned) [reply examples/dynamics/shot_at_an_angle_to_the_horizon.py:4]: Added in 6951f71

**Review by blackyblack — APPROVED** [review]

**Pull request merged**